ynh_systemd_action/ynh_systemd_action

```diff
@@ -28,32 +28,26 @@ ynh_systemd_action() {
     local log_path="${log_path:-/var/log/$service_name/$service_name.log}"
     local length=${length:-20}
     local timeout=${timeout:-300}
-    local wait_starting=true
 
-    if [[ -z "${line_match:-}" ]]
+    if [[ -n "${line_match:-}" ]]
     then
-        wait_starting=false
-    fi
-
-    ynh_clean_check_starting () {
-        # Stop the execution of tail.
-        kill -s 15 $pid_tail 2>&1
-        ynh_secure_remove "$templog" 2>&1
-    }
-
-    echo "Starting of $service_name" >&2
-    systemctl $action $service_name || ( journalctl --lines=$length -u $service_name >&2 && false)
-
-    if $wait_starting
-    then
-        # Following the starting of the app in its log
         local templog="$(mktemp)"
-        tail -F -n1 "$log_path" > "$templog" &
+	# Following the starting of the app in its log
+	if [ "$app_log" == "systemd" ] ; then
```

Comment:
I think we need to change `$app_log` to `$log_path` because it's not defined anywhere.

Reply:
Sorry, bad copy paste